### ISSUE TITLE
UI issue with Output view

### ISSUE THREAD

**Issue by jderieg** [AzBuilder/terrakube#619]
### Bug description 🐞

We are seeing an issue in the UI where _some_ (not all) workspaces throw an error when trying to view the Outputs tab (next to Resources in the Overview page). On some workspaces, this view works fine, but others will just go to a blank page like it's trying to load. The console shows the following React error:

![2023-12-04_11-47](https://github.com/AzBuilder/terrakube/assets/63372114/f2087bd0-d720-4caa-b39c-39392a54a3da)

I've tried with multiple browsers and they all throw the same error on the workspaces that have this issue. We're running Terrakube 2.18.1, but this issue was also present on 2.17.2.

### Steps to reproduce

Not sure what is different about these workspaces, so not exactly sure how to reproduce. Let me know if there is anything else I can look at to help troubleshooting.

### Expected behavior

Outputs for the state should show on the outputs tab

**Comment by jderieg** [comment]
I've been trying to figure out what could be causing it. Not seeing anything obvious. The dev console is showing that it's finding all 22 outputs on this state just fine, but then it throws that error.
![2023-12-04_16-19](https://github.com/AzBuilder/terrakube/assets/63372114/96665753-600e-41c0-9147-c45dd08fc24a)

**Comment by alfespa17** [comment]
I think I found the issue from your image, check line number 3 cluster_addons the type is an array of two and not a string like the others, I guess that when react component try to display the value is generating an error because the component is expecting a string value not an array, could you check other workspaces failing maybe you could find the same behavior.

**Comment by jderieg** [comment]
Yes, I'll give it a try this morning and let you know how it goes. Thanks!
